utils/geometry.go

```diff
@@ -55,22 +53,22 @@ func GetRandomDirection() Point {
 
 // Add add a given Point and returns the result
 func (p Point) Add(toAdd Point) Point {
-	return Point{X: p.X + toAdd.X, Y: p.Y + toAdd.Y}.Wrap()
```

Comment:
Hmm, I'd like to avoid having to remember to wrap my points around the grid every time I call Add- I also don't anticipate passing parameters into 'wrap' besides `GridUnitsWide` and `GridUnitsHigh`.

Reply:
Yeah, I don't love it either. I thought about adding a `PointManager` type that would contain the width and height and then all points it created would know how to wrap themselves. But then you can only use `Point`s that were created by the manager.

Another thought was to take the width and the height as parameters to `Add`, but then the function becomes kind of weird and harder to test.

Reply:
What if we define a private *Globals variable in the config package and provide a public function that allows setting it once (we could just error or return early if Set gets called a second time)? Then, we can have globally-accessible Getters and prevent the danger of having globally-editable constants. Or is that still gross?

(I made a PR to your branch to try it out: https://github.com/Zebbeni/protozoa/pull/15)